http-server/src/test/java/io/airlift/http/server/TestHttpServerProvider.java

```diff
@@ -371,6 +374,7 @@ public void testClientCertificateJava()
                 .setHttpsPort(0)
                 .setKeystorePath(getResource("clientcert-java/server.keystore").getPath())
                 .setKeystorePassword("airlift");
+        clientCertificate = ClientCertificate.REQUIRED;
```

Comment:
Make these `OPTIONAL` / `REQUESTED` to match the previous behavior. Or you could have the tests verify both modes work.

We should also add a test with `REQUIRED` where the client doesn't have a cert to confirm it fails. (since `REQUIRED` is a new mode that we didn't have before)